src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html

```diff
@@ -0,0 +1,38 @@
+<ct-modal-window modal-window-id="CARDS_PANEL">
+
+    <div class="ct-modal-window-bg" ct-modal-window-close></div>
+    <div class="ct-modal-window-content content">
+
+        <button ng-disabled="cards().knight===0">
```

Comment:
The buttons are dummies. When using of dev cards appears, new available actions will appear too, and the buttons will be enabled/disabled depend on them. For now, "disabling" using only for showing buttons' style